src/main/scala/org/allenai/pnp/semparse/SemanticParser.scala

```diff
@@ -268,13 +273,12 @@ class SemanticParser(val actionSpace: ActionSpace, val vocab: IndexedList[String
             entityBias <- param(SemanticParser.ENTITY_BIAS_PARAM + hole.t)
             entityWeights <- param(SemanticParser.ENTITY_WEIGHTS_PARAM + hole.t)
             entityChoiceScore = dot_product(entityWeights, rnnOutputDropout) + entityBias 
-            // TODO: How should we score these entities using attentions?
-            /*
+            // TODO: How should we score these entities using attentions
```

Comment:
I believe this version of entity scoring worked worse than the one below on GeoQuery. Something to keep in mind when we run experiments.